.goreleaser/preremove.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
 if nextdns status > /dev/null 2>&1; then
+    nextdns deactivate
```

Comment:
Uninstall does that already, see: https://github.com/nextdns/nextdns/blob/master/service.go#L200

We need to stop as well in the uninstall command.

Reply:
Yes Sir! You are Correct.